bin/serve

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+import sys
+import os
+import tensorflow_encrypted as tfe
+
+name = os.environ.get('TFE_NODE_NAME')
+server0_address = os.environ.get('TFE_SERVER0_ADDRESS')
+server1_address = os.environ.get('TFE_SERVER1_ADDRESS')
+producer_address = os.environ.get('TFE_CRYPTO_PRODUCER_ADDRESS')
+
+if name is None:
```

Comment:
It'd be nice if this was refactored a bit to be pushed as much as possible into `tensorflow_encrypted/config.py`.